examples/embedding/mpconfigport_minimal.h

```diff
@@ -132,3 +133,6 @@ typedef long mp_off_t;
 #else
 #include <alloca.h>
 #endif
+
+// Define some macroses
+#define NORETURN
```

Comment:
Should have a nawline at the end of the file

Reply:
done